src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,260 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
```

Comment:
Let's be careful with these.. these will also show up as Project Types in NPD.

We may want to add something like the following to the ide.host file so that we get the expected behavior in Visual Studio.  Note that for existing templates, we apply these overrides via some settings in VIsual Studio, but since this is a new template, we should do the right thing directly in the ide.host.json.

The following would make the tags consistent with the existing Blazor tag data we are overriding with in VS.

```

  "tags": [
    {
      "type": "projectType",
      "add": [ "Cloud", "Web" ],
      "remove": [ "*" ]
    },
    {
      "type": "platform",
      "add": [ "Linux", "macOS", "Windows" ]
    }
  ],
```